src/tests/class_tests/openms/source/SvmTheoreticalSpectrumGenerator_test.cpp

```diff
@@ -95,7 +95,13 @@ START_SECTION(void simulate(PeakSpectrum &spectrum, const AASequence &peptide, b
   PeakMap exp;
   MzMLFile mz_file;
 
+#if OPENMS_BOOST_VERSION_MINOR < 56
   mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test.mzML"),exp);
+  TEST_EQUAL(spec.size(), 7);
+#else
+  mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test_boost58.mzML"),exp);
+  TEST_EQUAL(spec.size(), 8);
+#endif
```

Comment:
it clearly uses libSVM and the RNG both: https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/CHEMISTRY/SvmTheoreticalSpectrumGenerator.cpp#L934 currently its not clear which one causes the issue